src/runtime/analytics-service.js

```diff
@@ -64,6 +68,7 @@ export class AnalyticsService {
 
     /**
      * @private @const {!TransactionId}
+     * TODO(diparikh): Remove once replaced by transactionId().
```

Comment:
It's already replaced. No?

Reply:
I mean to remove this and associated tests once this PR looks good.
Removed now.